core/src/sumeragi/mod.rs

```diff
@@ -342,18 +342,23 @@ pub const TELEMETRY_INTERVAL: Duration = Duration::from_secs(5);
 #[non_exhaustive]
 pub struct VotingBlock {
     /// At what time has this peer voted for this block
-    pub voted_at: Duration,
+    pub voted_at: Instant,
```

Comment:
This problem should go away after we use the formally verified replacement for `chrono`.

Reply:
there is no suggestion to edit here, right?